packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -105,6 +105,28 @@
   </screenshots>
 
   <releases>
+   <release version="2024.03.05-4" date="2024-03-05">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OSM maps data as of February 28, 2024</li>
+           <li>Export all bookmarks as a single KMZ file</li>
+         </ul>
+       <p>Linux Desktop:</p>
+         <ul>
+           <li>Fixed occasinal crash when there was no location service available</li>
```

Comment:
@rtsisyk a list of detailed desktop-related changes is usually provided to Ferenc- to add into the commit before the merge.

Reply:
We can still make changes here. It's retroactive anyway.